src/typechecker.erl

```diff
@@ -4012,12 +4011,12 @@ type_of_bin_element({bin_element, _P, Expr, _Size, Specifiers}, Source) ->
         [] when IsStringLiteral ->
             %% <<"abc">>
             type(string);
-        [] ->
+        [] when IsSigned ->
             %% <<X>>
```

Comment:
The comment should probably be `%% <<X/signed>>` in this case

Reply:
It's also signed if it's an expression but I'll put two comments basically

Reply:
Done